tests/test_aggregates.py

```diff
@@ -137,9 +140,42 @@ def test_disbursement_recipient(self):
             'cycle': 2012,
             'total': aggregate.total,
             'count': aggregate.count,
+            'memo_total': aggregate.memo_total,
+            'memo_count': aggregate.memo_count,
         }
         self.assertEqual(results[0], expected)
 
+    def test_disbursement_recipient_id_total(self):
+        committee = factories.CommitteeHistoryFactory(cycle=2012)
+
+        aggregate = factories.ScheduleBByRecipientIDFactory(
+            committee_id=committee.committee_id,
+            cycle=committee.cycle,
+            recipient_id='C00507368',
+            total=4000,
+            count=2,
+            memo_total=0,
```

Comment:
it's great to add a new test here - would factory values other than 0 be more likely to catch a mistake?